pkg/symbolizer/symbolizer.go

```diff
@@ -480,9 +480,6 @@ func (s *Symbolizer) symbolizeLocationsForMapping(ctx context.Context, m *pb.Map
 			if err := s.metadata.SetQuality(ctx, m.BuildId, dbginfo.Quality); err != nil {
 				return nil, nil, fmt.Errorf("set quality: %w", err)
 			}
-			if !dbginfo.Quality.HasDwarf && !dbginfo.Quality.HasGoPclntab && !(dbginfo.Quality.HasSymtab && dbginfo.Quality.HasDynsym) {
```

Comment:
It'll be still nice to keep this check. We can do the same as you've done at other places - we can check for either .symtab or .dynsym instead of both.